lib/product-import.coffee

```diff
@@ -210,8 +210,11 @@ class ProductImport
       when 200 then @_summary.updated++
 
   _createProductFetchBySkuQueryPredicate: (skus) ->
-    skuString = "sku in (#{skus.map((val) -> JSON.stringify(val))})"
-    return "masterVariant(#{skuString}) or variants(#{skuString})"
+    if skus.length is 0
+      'masterVariant(sku in ("")) or variants(sku in (""))'
```

Comment:
I created the follow up issue [here](https://github.com/sphereio/sphere-product-import/issues/106)